metabase/metabase
# dynamic waterfall

Closes https://github.com/metabase/metabase/issues/35820

### Description

Creates the dynamic component for the echarts waterfall chart. Note that not all functionality is working yet, for example the tooltip and click actions menu don't appear for the total bar. Missing functionality will be added in later PRs.

### How to verify

1. Create a question with a waterfall chart
2. Should render correctly in QB
3. Add it to a dashboard, it should render correctly there as well

### Demo

https://github.com/metabase/metabase/assets/37751258/304b1305-0acb-4db1-bd32-bae9fdf54746
